packages/http-server-csharp/test/diagnostic.test.ts

```diff
@@ -0,0 +1,110 @@
+import { BasicTestRunner, expectDiagnostics } from "@typespec/compiler/testing";
+import { beforeEach, describe, it } from "vitest";
+import { createCSharpServiceEmitterTestRunner, getStandardService } from "./test-host.js";
+describe("http-server-csharp: diagnostics", () => {
```

Comment:
don't need to have top level describe with vitest just adds an extra nesting in the tree explorer